src/libraries/Common/src/System/Experimentals.cs

```diff
@@ -0,0 +1,25 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System
+{
+    internal static class Experimentals
+    {
+        internal const string SharedUrlFormat = "https://aka.ms/dotnet-warnings/{0}";
+
+        // Please see docs\project\list-of-diagnostics.md for instructions on the steps required
+        // to introduce an experimental API, claim a diagnostic id, and ensure the
+        // "aka.ms/dotnet-warnings/{0}" URL points to documentation for the API.
+        // The diagnostic IDs reserved for experimental APIs are SYSLIB5### (SYSLIB5001 - SYSLIB5999).
+
+        // When an API is no longer marked as experimental, the diagnostic ID should be removed from this file
```

Comment:
If we're using the markdown file as the source of truth for claiming new diagnostic IDs, what's the need for also consolidating them in source? IIRC we don't do this for other diagnostics.

Reply:
We do this with Obsoletions as well. I assert that it has helped keep the process discoverable and consistent.